images/dog-list.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Home Page</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width-device-width; initial-scale=1.0;">
+        <style>
+            
+            <link rel="stylesheet"
+                href="https://fonts.googleapis.com/css?family=Open+Sans|Raleway&display=swap"> 
+            
+            *       { box-sizing: border-box; }
+            body    { margin: 0 auto; max-width: 100%; }    
+            img     { width: 100%; }
+            h1, h2, h3, h4 { font-family: 'Open Sans', sans-serif; }
+            p       { font-family: 'Raleway', sans-serif; }
+            button  { font-family: 'Raleway', sans-serif; }
+        
+        </style>
+    
+    </head>
+    <body>
+        <header style="background-image: url('header-image.jpg')">
+            <section>
+                <!--Logo here-->
+                <img src="adoptadog-logo.svg" alt="Adoptadog">
+                <h1>Seattle's premiere dog adoption agency</h1>
+            </section>
+            <section>
+                <!--Cart here-->
+                <img src="cart-icon.svg" alt="Your Cart">
+                <p>$1234.56</p>
+            </section>
+            <button>
+                Checkout
+            </button>
+        </header>
+        <nav>
+            <ul>
+                <li> <a href="index.html">Home</a> </li>
+                <li> <p>/</p> </li>
+                <li> <a href="dog-list.html">Dogs</a> </li>
+                <li> <p>/</p> </li>
+                <li> <a href="blog.html">Blog</a> </li>
+            </ul>
+        </nav>
+        <main>
+            <section>
+                <img src="home-intro.jpg" alt="Happy Dog"> 
+                <h2> Adopt A Dog </h2>
+                <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. </p>
+            </section>
+            <section>
+                <div class="flex-box">
+                    <article>
+                        <img src="murphy-card.jpg" alt="Murphy"> 
+                        <h3>Murphy</h3>
+                        <h4>Cost To Adopt:</h4>
+                        <p>$123.45</p>
+                        <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. </p>
+                        <button>
+                            Adopt
+                        </button>
+                    </article>
+                    <article>
+                        <img src="poppy-card.jpg" alt="Poppy"> 
```

Comment:
I'd encourage you to move your images into an image folder, just for easier organization